packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -104,6 +108,7 @@ def create_pool(
             syft_worker_image_id=image_uid,
             max_count=number,
             workers=workers,
+            syft_worker_image_name_tag=worker_image.full_tag,
```

Comment:
MAYBE: We could set the image tag to `worker_image.image_tag`

Reply:
Done. Now we pass `worker_image: SyftWorkerImage` into `WorkerPool`